### ISSUE TITLE
[BUG] Filter out igb error in Up Extreme suspend-resume tests

### ISSUE THREAD

**Issue by plbossart** [thesofproject/sof-test#607]
````
===========================>>
[ 2256.981433] kernel: igb 0000:01:00.0 enp1s0: Reset adapter
<<===========================
````
That's annoying to use a USB dongle on a board that has 2 Ethernet ports...

**Comment by plbossart** [comment]
This should work?

````
ignore_str="$ignore_str"'|igb 0000:0.:00.0 enp.s.: Reset adapter
````

**Comment by marc-hb** [comment]
@plbossart  closed by #631 ?
